sky/core.py

```diff
@@ -51,6 +51,40 @@ def status(refresh: bool = False) -> List[Dict[str, Any]]:
             'metadata': (dict) metadata of the cluster,
         }
 
+    Each cluster can have one of the following statuses:
+
+    - ``INIT``: The cluster may be live or down. It can happen in the following
```

Comment:
Not sure if it is my laptop's rendering problem, but the `INIT`/`UP`/`STOPPED` looks bolded in the `Python API` page, but normal in the `Command Line Interface` page.